src/lib/server/amplitude.ts

```diff
@@ -0,0 +1,12 @@
+import { dev } from '$app/environment';
+import { track } from '@amplitude/analytics-node';
+
+export function trackEvent(
+	name: string,
+	eventProperties: Record<string, unknown> | undefined,
```

Comment:
you can make this optional and the last parameter, then when calling the function you can omit the parameter if we don't pass it in the calls.

Reply:
by optional I mean doing `eventProperties?:`

Reply:
Does not work exactly like that have to swap the params but I think still better.